modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/utilities/NetCDFCRSUtilities.java

```diff
@@ -123,6 +123,14 @@
 
     private static final Map<AxisType, String> OPPOSITES = new HashMap<AxisType, String>(16);
 
+    /**
+     * this flag states if an automatic conversion from km to m should happen with axis/coordinates
+     */
+    public static final String CONVERT_AXIS_KM_KEY =
```

Comment:
You said in other parts of the code that the conversion could be due to using feet... so why make it specific to KM?